<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>501</width>
     <height>600</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="layout">
    <item row="0" column="0" colspan="3">
     <widget class="QTextBrowser" name="responseBrowser">
      <property name="minimumSize">
       <size>
        <width>499</width>
        <height>350</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>499</width>
        <height>350</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="whatsThis">
       <string>Response from server</string>
      </property>
      <property name="undoRedoEnabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QCommandLinkButton" name="closeBtn">
      <property name="text">
       <string>Close</string>
      </property>
      <property name="icon">
       <iconset resource="res/res.qrc">
        <normaloff>:/img/img/close.png</normaloff>:/img/img/close.png</iconset>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <spacer name="spaser">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="QPlainTextEdit" name="requestEditor">
      <property name="minimumSize">
       <size>
        <width>499</width>
        <height>150</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>499</width>
        <height>150</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="tabChangesFocus">
       <bool>true</bool>
      </property>
      <property name="plainText">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Write a request here...</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QCommandLinkButton" name="sendBtn">
      <property name="text">
       <string>SEND</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QCommandLinkButton" name="clearBtn">
      <property name="text">
       <string>Clear</string>
      </property>
      <property name="icon">
       <iconset resource="res/res.qrc">
        <normaloff>:/img/img/clear_left.png</normaloff>:/img/img/clear_left.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res/res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>clearBtn</sender>
   <signal>clicked()</signal>
   <receiver>requestEditor</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>376</x>
     <y>552</y>
    </hint>
    <hint type="destinationlabel">
     <x>185</x>
     <y>409</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>closeBtn</sender>
   <signal>clicked()</signal>
   <receiver>Widget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>388</x>
     <y>569</y>
    </hint>
    <hint type="destinationlabel">
     <x>287</x>
     <y>596</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearBtn</sender>
   <signal>clicked()</signal>
   <receiver>responseBrowser</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>432</x>
     <y>535</y>
    </hint>
    <hint type="destinationlabel">
     <x>423</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
